src/main/repository/UserRepository.ts

```diff
@@ -1,16 +1,26 @@
-import { EntityRepository, Repository } from 'typeorm';
+import {
+  DeleteResult,
```

Comment:
Could you also remove some of the unused imports? I think there are warnings rn for DeleteResult, UpdateResult, and UserUpdateDTO :)

Reply:
Oh I forgot to remove these, thanks!